pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -387,7 +387,11 @@ func (m *cgroupManagerImpl) toResources(resourceConfig *ResourceConfig) *libcont
 	// for each page size enumerated, set that value
 	pageSizes := sets.NewString()
 	for pageSize, limit := range resourceConfig.HugePageLimit {
-		sizeString := units.CustomSize("%g%s", float64(pageSize), 1024.0, libcontainercgroups.HugePageSizeUnitList)
+		sizeString, err := v1helper.HugePageUnitSizeFromByteSize(pageSize)
+		if err != nil {
```

Comment:
if the page size was wrong, it would error on the runc side as well.

this warning seems primarily useful for developer debugging.  the api server validation should prevent this error case from ever happening, so this is fine.